debug/line-interactivity.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```

Comment:
@davidmanzanares I have seen something weird with this example, regarding to 'Interactivity'. See this gif:

![interactivity-issue](https://user-images.githubusercontent.com/458196/46340708-8b282e00-c636-11e8-87d6-6794a1a8bfa9.gif)

It's like the coordinates have a strange offset

Reply:
I didn't record it, but that also happens when using tilt

Reply:
Could you re-check that? I'm not seeing the same behavior:
![peek 2018-10-02 12-10](https://user-images.githubusercontent.com/1692175/46342910-5dde7e80-c63c-11e8-9b57-867d69015010.gif)

Reply:
I'm still having issues. I'm using Chrome v 69.0.3497.100 on Mac OS X 10.13.6

Reply:
Debugging it with @rochoa showed that this is probably caused by high-DPI screens with high devicePixelRatios. I'm looking for a fix now.